README.rst

```diff
@@ -50,7 +50,8 @@ DNS lookup...
   the lookup.
 * If ``gethostbyname`` does not have it cached then a request is made to the
   known DNS server that was given to the network stack. This is typically the
-  local router or the ISP's caching DNS server
+  local router or the ISP's caching DNS server.
+* Port 53 is opened to send UDP request to DNS server.
```

Comment:
From wikipedia: `The Transmission Control Protocol (TCP) is used when the response data size exceeds 512 bytes, or for tasks such as zone transfers. Some resolver implementations use TCP for all queries.`

I think we should at least mention something like `(if the response size is too large, TCP will be used instead).`